.github/workflows/java-ci.yml

```diff
@@ -77,7 +77,7 @@ jobs:
     - uses: actions/setup-java@v1
       with:
         java-version: 8
-    - run: ./gradlew -DflinkVersions=1.12,1.13,1.14 -DsparkVersions=2.4,3.0,3.1,3.2 -DhiveVersions=2,3 build -x test -x javadoc -x integrationTest
+    - run: ./gradlew -DflinkVersions=1.12,1.13,1.14 -DsparkVersions=2.4,3.0,3.1,3.2 -DscalaVersion=2.12 -DhiveVersions=2,3 build -x test -x javadoc -x integrationTest
```

Comment:
Can't this be the default Scala version so we don't need to set it here?

Reply:
removed